xde/src/xde.rs

```diff
@@ -1097,8 +1112,9 @@ unsafe extern "C" fn xde_detach(
                 // We explicitly drop them in order here to ensure there are no
```

Comment:
Update comment to also indicate `MacUnicastHandle` has a ref on `MacClientHandle` as well?

Reply:
Done